pacman/operations/rigged_algorithms/basic_test.py

```diff
@@ -0,0 +1,49 @@
+from spinn_utilities.progress_bar import ProgressBar
+
+# pacman imports
+from pacman.utilities.algorithm_utilities import placer_algorithm_utilities
+from pacman.model.placements import Placement, Placements
+from pacman.utilities.utility_objs import ResourceTracker
+
+# general imports
+import logging
+logger = logging.getLogger(__name__)
+
+
+class BasicPlacerTest(object):
```

Comment:
what does this class do exactly? it looks like the basic placer but labelled test. if so, couldnt it just be removed?